.example/tests/integration/charms/k8s/package/README.md

```diff
@@ -0,0 +1 @@
+../../../../../README.md
```

Comment:
Do we really need the README in the integration testing version?

Reply:
Ah, I see. Up to you. Without the symlink is slightly simpler, but with it is more consistent.